config/crds/network.kubesphere.io_ippools.yaml

```diff
@@ -0,0 +1,121 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: ippools.network.kubesphere.io
+spec:
+  group: network.kubesphere.io
+  names:
+    kind: IPPool
+    plural: ippools
+  scope: Cluster
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            blockSize:
+              description: The block size to use for IP address assignments from this
+                pool. Defaults to 26 for IPv4 and 112 for IPv6.
+              type: integer
+            cidr:
+              description: The pool CIDR.
+              type: string
+            disabled:
+              description: When disabled is true, Calico IPAM will not assign addresses
```

Comment:
We need to change all `Calico IPAM` to ower selves.

Reply:
Done. I looked,  that's the only thing I haven't changed.